packages/webawesome/src/components/checkbox/checkbox.css

```
@@ -45,6 +45,9 @@
   vertical-align: middle;
   cursor: pointer;
 }
+[part~='label'] {
+  display: inline-block;
```

```suggestion
  display: inline;
```

We use `inline` for `wa-radio`'s labels, so we may want to make this one consistent.